privval/signer_dialer_endpoint.go

```diff
@@ -240,3 +204,42 @@ func (ss *SignerDialerEndpoint) handleRequest() {
 func (ve *SignerDialerEndpoint) isConnected() bool {
 	return ve.IsRunning() && ve.conn != nil
 }
+
+func (ss *SignerDialerEndpoint) serviceLoop() {
+	defer close(ss.stoppedCh)
+
+	retries := 0
+	var err error
+
+	for {
+		select {
+		default:
+			{
```

Comment:
Nit: the extra `{}` for each case make this extra indented for no reason.

Reply:
fixed